### PR TITLE
[9.18] Time entry listing and submission for merge requests

### LINKED ISSUE #421: No time entry support for merge requests
No support for listing MR's time entries and updating them

### PR DESCRIPTION
**Pull request by Cysioland**
Probably fixes #421

### CODE REVIEW

File: lib/Gitlab/Api/MergeRequests.php [hunk 1]
```diff
@@ -373,4 +373,26 @@ public function awardEmoji($project_id, $mr_iid)
...
+     */
+    public function getTimeStats($project_id, $mr_iid)
+    {
+        return $this->get($this->getProjectPath($project_id, 'merge_requests/'.$this->encodePath($mr_iid) .'/time_stats'));
+    }
+
+    /**
+     * @param $project_id
+     * @param $mr_iid
+     * @param $duration
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/MergeRequests.php:390]: Invalid syntax. You must include types.

**Comment by GrahamCampbell** [comment]
Thanks for starting work on this @Cysioland. Are you able to finish this PR? 🚀

**Comment by Cysioland** [comment]
Not in this very moment, but I will

**Comment by GrahamCampbell** [comment]
Excellent. :)

**Comment by GrahamCampbell** [comment]
Oh, it looks like this is already implemented on the 9.18 branch. Thanks anyway. :)

**Pull request closed without merging**